rllib/algorithms/es/es_tf_policy.py

```diff
@@ -148,10 +149,21 @@ def __init__(self, obs_space, action_space, config):
         )
 
     @override(Policy)
-    def compute_actions(self, observation, add_noise=False, update=True, **kwargs):
+    def compute_actions(self, obs_batch=None, add_noise=False, update=True, **kwargs):
```

Comment:
This unifies the signature with the rest of RLlib so we can make proper use of it with kwargs